doc/manual/src/architecture/architecture.md

```diff
@@ -19,15 +19,15 @@ Nix consists of hierarchical [layers](https://en.m.wikipedia.org/wiki/Multitier_
 |                               | evaluates to                    |
 |                               |                                 |
 |               referenced by   V       builds                    |
-|   [ build input ] ----> [ build plan ] ----> [ build result ]   |
+|     [ build input ] --> [ build task ] --> [ build result ]     |
 |                                                                 |
```

Comment:
My arrow is very ugly :weary:! But should we try to indicate that build tasks can depend on other build tasks?
```suggestion
|                             \_^                                 |
|                                                                 |
```

Reply:
For the very top-level introduction - no. I suppose once we get into sufficient level of detail to introduce these concepts we will also have thought about tooling to display proper diagrams.